parsigs/parse_sig_api.py

```diff
@@ -344,10 +358,43 @@ def _get_times_from_latin(frequency):
         return latin_freq.times
 
 
+def _get_interval_from_latin(frequency):  # yoad added this for cases like qxd/qxh (the interval changes and times is 1)
```

Comment:
No need to add comments describing who did the change - this is understood by git annotate